frontend/package.json

```diff
@@ -43,12 +43,15 @@
     "yarn": ">= 1.22.5"
   },
   "devDependencies": {
+    "@apollo/react-testing": "^4.0.0",
```

Comment:
**nitpick**: we use exact versions in this project, but this will be picked up by renovate.